tests/test-definition.yaml

```diff
@@ -15,13 +15,14 @@ dimensions:
     values:
       - "451"
       - "455"
+      - "469"
```

Comment:
Please grep once through the repo and add 470 where applicable.

Reply:
Done in 46b26f1